prow/config/prow-config-documented.yaml

```diff
@@ -904,6 +904,31 @@ plank:
 pod_namespace: ' '
 
 
+# ProwJobDefaultEntries holds a list of defaults for specific values
+# Each entry in the slice specifies Repo and CLuster regexp filter fields to
+# match against the jobs and a corresponding ProwJobDefault . All entries that
+# match a job are used. Later matching entries override the fields of earlier
+# matching entires.
+prowjob_default_entries:
+  - # Cluster matches against the cluster alias of the build cluster that the
+    # ProwJob is configured to run on. Recall that ProwJobs default to running on
+    # the "default" build cluster if they omit the "cluster" field in config.
+    cluster: ' '
+
+    # Config is the ProwJobDefault to apply if the filter fields all match the
+    # ProwJob. Note that when multiple entries match a ProwJob they are all used
+    # by sequentially merging with later entries overriding fields from earlier
+    # entries.
+    config:
+        tenant_id: ' '
+
+    # OrgRepo matches against the "org" or "org/repo" that the presubmit or postsubmit
+    # is associated with. If the job is a periodic, extra_refs[0] is used. If the
+    # job is a periodic without extra_refs, the empty string will be used.
+    # If this field is omitted all jobs will match.
+    repo: ' '
```

Comment:
No interest in `repos` or `clusters`? It seems like this will be a common scenario.

Reply:
Same as above. I want to keep this consistent with `default_decoration_config_entries` There is already a fine way to do this as you can have multiple entries with the same tenant_id, but I don't believe this will be that common of a usecase.

Reply:
+1 for consistency. Additional matching fields can be added as needed in the future with this design too.